api/config/environments/development.rb

```diff
@@ -14,16 +14,16 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join("tmp", "caching-dev.txt").exist?
-    config.cache_store = :memory_store
-    config.public_file_server.headers = {
-      "Cache-Control" => "public, max-age=#{2.days.to_i}"
-    }
-  else
-    config.action_controller.perform_caching = false
-
-    config.cache_store = :null_store
-  end
+  #if Rails.root.join("tmp", "caching-dev.txt").exist?
```

Comment:
Should we just remove these lines instead of commenting?

Reply:
Yes, I meant to remove those actually, unless you don't want to have caching in development. I think it's beneficial, though.